site/content/en/docs/concepts/elastic_workload.md

```diff
@@ -0,0 +1,71 @@
+
+title: "Elastic Workloads"
+date: 2025-04-16
+weight: 4
+description: >
+  Workload types that support dynamic scaling.
+
+
+{{< feature-state state="alpha" for_version="v0.13" >}}
+
+## Elastic Workloads (Workload Slices)
+
+Elastic Workloads extend the core `Workload` abstraction in Kueue to support **dynamic scaling** of admitted jobs, without requiring suspension or requeueing.
+This is achieved through the use of **Workload Slices**, which track partial allocations of a parent job's scale-up and scale-down operations.
+
+This feature enables more responsive and efficient scheduling for jobs that can adapt to changing cluster capacity, particularly in environments with fluctuating workloads or constrained resources.
+
+## Dynamic Scaling
+
+Traditionally, a `Workload` in Kueue represents a single atomic unit of admission.
+Once admitted, it reflects a fixed set of pod replicas and consumes a defined amount of quota. If a job needs to scale up or down, the `Workload` must be suspended, removed, or replaced entirely.
+
+While scaling **down** a workload is relatively straightforward and does not require additional capacity or a new workload slice, scaling **up** is more involved. It requires *additional capacity* that must be explicitly requested and *admitted* by Kueue through a new `Workload Slice`.
```

Comment:
I think somewhere in this section we should add a note that the scaling operation assumes workload to remain in the same ResourceFlavor.

Reply:
Do you think adding this to the "Limitations" section would be a good choice?

Reply:
Updated "Limitations" - PTAL.